### ISSUE TITLE
TypeError: Cannot set property 'file_name' of undefined

### ISSUE THREAD

**Issue by zipang** [vimeo/vimeo.js#65]
When calling `replace` method with these params : 

```
vimeoClient.replace(localPath, uri, callback, progressReport, errorReport)
```

The culprit is setting `params` to undefined at line 512 of vimeo.js : 
```
params = undefined
```
and then trying to set `params.file_name`

**Comment by ArvinH** [comment]
Same as `upload` method.

```
vimeoClient.upload(localPath, uri, callback, progressReport, errorReport)
```

replace `params` to `undefined` in line 432 of vimeo.js
try to read `params.upload` in line 450

**Comment by erunion** [comment]
Looks like https://github.com/vimeo/vimeo.js/pull/66 will fix this.

**Comment by erunion** [comment]
Fix tagged to 2.0.1. Thanks @ArvinH!
